js/main.js

```diff
@@ -1761,14 +1825,20 @@ function BlackboxLogViewer() {
                     case "9".charCodeAt(0):
                         try {
                         	if(!e.altKey) { // Workspaces feature
+                                var id = e.which - 48
                         		if (!e.shiftKey) { // retreive graph configuration from workspace
-		                            if (workspaceGraphConfigs.graphConfig[e.which-48] != null) {
-		                                newGraphConfig(workspaceGraphConfigs.graphConfig[e.which-48]);
+                                    if (workspaceGraphConfigs[id] != null) {
```

Comment:
This indentation is totally out, but I suppose is because we have lines with tabs and others with spaces :(

Reply:
I'll fix. Which do we prefer?

Reply:
Seems spaces are used.

Reply:
Yes, we are trying to use 4 spaces, but there are a lot tabs from older files...